src/App.tsx

```diff
@@ -1,61 +1,100 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 
 export const App = () => {
+  const [text, setText] = useState('');
```

Comment:
`title` will be more descriptive in my opinion
```suggestion
  const [title, setTitle] = useState('');
```